Examples/TcpReassembly/main.cpp

```diff
@@ -329,7 +328,7 @@ void listInterfaces()
 
 	std::cout << std::endl << "Network interfaces:" << std::endl;
 
-	for (const auto& interface : devList)
+	for (auto& interface : devList)
```

Comment:
@seladb The problem is still there. How do you check if the code compiles for different systems? Using Docker or directly on GitHub with its fast CI? I need a faster way to test changes; right now, I can test only 2 or 3 times in an hour.

Reply:
I usually use VMs to test on different platforms. For unix/linux-based platforms you can also use docker.

For Windows you can use these free VMs offered by Microsoft:
https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/

You can install VS 2019 / 2022 and test whatever you need

Reply:
@jpcofr I think the issue is with the variable name... `interface` is probably a reserved word in Visual Studio 🤦‍♂️ 

I changed `interface` to `dev` and it seems to work now...